R/friedman.qmd

```diff
@@ -0,0 +1,43 @@
+## title: "friedman's test"
```

Comment:
In a quarto file the the page should begin with the following format with just three dashes to denote the beginning and end:
# ---
# title: "Friedman's Test"
# ---